onnxruntime/core/optimizer/conv_activation_fusion.cc

```diff
@@ -84,14 +84,15 @@ class ConvActivationSelector : public NodeSelector {
     }
 
     auto is_supported_non_cuda_rocm_ep_activation = [&graph_viewer](const Node& activation_node) {
-      if (graph_utils::IsSupportedOptypeVersionAndDomain(activation_node, "Relu", {6, 13, 14}) ||
-          graph_utils::IsSupportedOptypeVersionAndDomain(activation_node, "Sigmoid", {6, 13}) ||
-          graph_utils::IsSupportedOptypeVersionAndDomain(activation_node, "Tanh", {6, 13}) ||
-          graph_utils::IsSupportedOptypeVersionAndDomain(activation_node, "LeakyRelu", {6, 16})) {
+      const auto& domain = activation_node.Domain();
+      if (graph_utils::IsSupportedOptypeVersionAndDomain(activation_node, "Relu", {6, 13, 14}, domain) ||
+          graph_utils::IsSupportedOptypeVersionAndDomain(activation_node, "Sigmoid", {6, 13}, domain) ||
+          graph_utils::IsSupportedOptypeVersionAndDomain(activation_node, "Tanh", {6, 13}, domain) ||
+          graph_utils::IsSupportedOptypeVersionAndDomain(activation_node, "LeakyRelu", {6, 16}, domain)) {
```

Comment:
I don't think adding domain to these checks should be necessary. We only use the internal NHWC domain for layout sensitive operators so none of the activations/Add/Clip would ever be in that domain.